pkg/service/utgen/assets/test_generation.toml

```diff
@@ -65,13 +74,21 @@ The following packages are already installed in the environment. Use these when
 {{ .installed_packages | trim }}
 =========
 
-{{ if .additional_includes_section | trim }}
+
+{%- if additional_includes_section | trim %}
 {{ .additional_includes_section | trim }}
-{{ end }}
+{% endif %}
+
```

Comment:
i removed endif 
<img width="1290" alt="image" src="https://github.com/user-attachments/assets/b06ac1c3-1e23-4079-85f5-b8f5e5a33a22">
don't put it back